django_project/monitor/models.py

```diff
@@ -101,6 +102,23 @@ def save(self, *args, **kwargs):
             except AttributeError:
                 pass
 
+        validate_ocean = kwargs.pop('validate_ocean', False)
+        if validate_ocean:
```

Comment:
@zamuzakki can you add exception handling in case of network issues or a non-200 response?

Reply:
@dimasciput updated